packages/fxa-auth-server/test/bench/index.js

```diff
@@ -13,7 +13,7 @@ const through = require('through');
 let clientCount = 2;
 const pathStats = {};
 let requests = 0;
-let pass = 0; // eslint-disable-line no-unused-vars
+let pass = 0; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
According to my local checkout of this branch, we don't need this eslint ignore line?

Reply:
Okay we tried this locally, and while only one eslint rule is needed, eslint still marks them both as errors in the IDE. We'll just leave them both there as they're innocuous.